test/cpp/util/tls_test_utils.h

```diff
@@ -76,6 +76,25 @@ class AsyncCertificateVerifier
   std::deque<Request> queue_ ABSL_GUARDED_BY(mu_);
 };
 
+class VerifiedRootCertSubjectVerifier
+    : public grpc::experimental::ExternalCertificateVerifier {
+ public:
+  explicit VerifiedRootCertSubjectVerifier(char* expected_subject)
```

Comment:
Can we pass in an `absl::string_view` (preferred) or a `const std::string&` instead?

Reply:
Will change